File: ObjCSample/SampleApp/StoryboardNextViewController.m
#import "StoryboardNextViewController.h"

@interface StoryboardNextViewController ()
@property (nonatomic, strong) IBOutlet UIToolbar *toolbar;
@end

@implementation StoryboardNextViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    self.view.backgroundColor = self.backgroundColor;
    self.toolbar.hidden = self.hideToolbar;
}

- (IBAction)cancel
{
    [self dismissViewControllerAnimated:YES completion:self.viewControllerDismissedCompletion];
}

@end

File: SwiftSamplePackage/SampleApp/StoryboardNextViewController.swift
import UIKit

final class StoryboardNextViewController: UIViewController {
    var backgroundColor: UIColor!
    var hideToolbar = true
    var viewControllerDismissedCompletion: (() -> Void)?
    @IBOutlet private(set) var toolbar: UIToolbar!
    @IBOutlet private(set) var cancelButton: UIBarButtonItem!

    override func viewDidLoad() {
        super.viewDidLoad()
        view.backgroundColor = backgroundColor
        toolbar.isHidden = hideToolbar
    }

    @IBAction private func cancel() {
        dismiss(animated: true, completion: viewControllerDismissedCompletion)
    }
}
